response_operations_ui/static/js/read-csv.js

```diff
@@ -20,7 +10,7 @@ function drawOutput(lines){
 	preview = preview + "    <h3 class='venus'>Sample contents</h3>";
 	preview = preview + "  </div>";
 	preview = preview + "  <div class='panel__body'>";
-	preview = preview + "    <div id='sample-preview-businesses'>Number of businesses: " + lines.length + "</div>";
+	preview = preview + "    <div id='sample-preview-businesses'>Number of businesses: " + businessCount + "</div>";
```

Comment:
At some point this should be changed to toggling the visibility of some html rather than directly rendering from javascript.

Reply:
Not saying to do it now, just a comment.